omod/src/main/java/org/openmrs/module/coreapps/page/controller/MergeVisitsPageController.java

```diff
@@ -15,26 +15,38 @@
 import org.openmrs.ui.framework.annotation.SpringBean;
 import org.openmrs.ui.framework.page.PageModel;
 import org.openmrs.ui.framework.page.Redirect;
+import org.springframework.util.StringUtils;
```

Comment:
Could there be a reason why you have not used StringUtils from Apache Commons?

Reply:
Good catch, i will change it.